tests/Fixtures/Integration/priority/ordered_class_elements,class_attributes_separation.test

```diff
@@ -1,7 +1,7 @@
 --TEST--
-Integration of fixers: ordered_class_elements,method_separation.
```

Comment:
do we keep those for the other deprecated fixers? seems like a lot of copy-paste for not much value to me...

Reply:
This has value: deprecating a fixer does not mean we allow it to break. If we removed tests for other deprecated fixers, that's a mistake IMO.

Reply:
Having the fixture test `A+B` and `C+B` where `A === C` has little value, as we know the `A === C` because we test this as `A` is now proxied to `C`, no?

Reply:
`A` being proxied to `C` is an implementation detail. The test is here to proove `A` continues to work as expected, no matter how it is implemented. Testing `C` does not mean `A` uses it correctly.

Reply:
I agree with @julienfalque here
it's extra easy to drop tests for deprecated fixer when it got removed. we keep tests for other deprecated fixers. 
imagine that in future one will change ProxyFixer implementation, or A is proxied to B with configuration. we need to know it works the same as it was, so we change implementation, but keep tests.

Reply:
so IIUC, I should restore the old fixtures-files and than copy-paste those using the new fixer name and configuration so we've both?

Reply:
Yes.

Reply:
and how to deal with deprecation notices raised in the integration tests than?

Reply:
Tests with deprecated fixers should be moved to a specific method in the test class with a `@group legacy` annotation. But I think fixers don't trigger deprecation notices so you might not have to change anything. Though we should take care of it in a separate PR.